src/components/Admin/AdminHashtags/AddHashtagDialog.tsx

```diff
@@ -0,0 +1,95 @@
+import { useAppDispatch, useAppSelector } from '@/hooks/common';
+import { IError } from '@/interfaces/common';
+import { postHashTagThunk, selectHashTags } from '@/reducers/hashTagsSlice';
+import { notifySuccess } from '@/reducers/notificationSlice';
+import { getErrorText } from '@/utils/validation';
+import { Button, Dialog, TextInput } from 'hds-react';
+import { ChangeEvent, FC, useCallback, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface IAddhashtagDialogProps {
+  isOpen: boolean;
+  onToggleAddHashtagDialog: () => void;
+}
+
+const AddHashtagDialog: FC<IAddhashtagDialogProps> = ({ isOpen, onToggleAddHashtagDialog }) => {
+  const { t } = useTranslation();
+  const dispatch = useAppDispatch();
+  const { Header, Content, ActionButtons } = Dialog;
+  const [hashtagName, setHashtagName] = useState('');
+  const titleId = 'add-hashtag-title';
```

Comment:
titleId and descriptionId are immutable values so they can me moved outside the component